scipy/interpolate/interpolate.py

```diff
@@ -623,17 +623,16 @@ def _check_bounds(self, x_new):
 
 
 class _PPolyBase(object):
-    """
-    Base class for piecewise polynomials.
-    """
+    """Base class for piecewise polynomials."""
     __slots__ = ('c', 'x', 'extrapolate', 'axis')
 
-    def __init__(self, c, x, extrapolate=None, axis=0):
```

Comment:
The `None` default value sentinel is intentional, and I would prefer to retain it. The reason for it is to push the definition of the default value to the base class, so that you don't need to repeat the default (and remember what it is) e.g. in subclass definitions, unless there is some reason to remember it. Moreover, it can be useful to be able to call `extrapolate=None` to indicate that the default behavior should be used.